server/models/User.js

```diff
@@ -267,4 +270,23 @@ User.afterCreate(async (user, options) => {
   ]);
 });
 
+User.prototype.getCount = async function () {
+  const countSql = `
+    SELECT 
+      COUNT(CASE WHEN "suspendedAt" IS NOT NULL THEN 1 END) as "suspendedCount",
+      COUNT(CASE WHEN "isAdmin" = true THEN 1 END) as "adminCount",
+      COUNT(CASE WHEN "lastActiveAt" IS NULL THEN 1 END) as "invitedCount",
+      COUNT(CASE WHEN "suspendedAt" IS NULL AND "lastActiveAt" IS NOT NULL THEN 1 END) as "activeCount",
+      COUNT(*) as count
+    FROM users
+    WHERE "deletedAt" IS NULL
```

Comment:
Your missing a critical part of the `WHERE` query, need to pass in and include the `teamId` otherwise we're counting the entire userbase.